SparkPost/2web2ui
# FE-884 - Component Wrapper for Matchbox ProgressBar

FE-884 - Component Wrapper for Matchbox ProgressBar

### What Changed
 - ProgressBar has a single, globally accessible point of entry within the app
 - ProgressBar is only imported from Matchbox in one place - replaced all instances of the ProgressBar directly imported from matchbox with this new global component
- New props available to the Hibana version of the component are exposed - and subsequently ignored by the original component (https://jira.int.messagesystems.com/browse/FE-812)
- Ensure no unusual positioning UI defects are introduced as a result of this change - both for when using Hibana and when using the default theme

### How To Test
 - enable hibana ui flag 
`{
	"options": {
		"ui":{
"hibana":{
	"hasThemeControls": true
}
		}
	}
 }`
- Check all instances of ProgressBar for ui defects with and without hibana enabled

### To Do
